nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/trash/AbstractTrashService.java

```diff
@@ -58,6 +59,8 @@
  */
 public abstract class AbstractTrashService implements TrashService {
 
+    public static final String SEPARATOR = "/";
```

Comment:
Could you rename it `PATH_SEPARATOR` and make it protected? Because it's not the responsibility of TrashService to expose the path seperator.